packages/mobile/src/components/lineup/Lineup.tsx

```diff
@@ -403,17 +409,20 @@ export const Lineup = ({
       ListFooterComponent={<View style={{ height: 16 }} />}
       onEndReached={handleLoadMore}
       onEndReachedThreshold={LOAD_MORE_THRESHOLD}
-      // TODO: Either style the refreshing indicator or
-      // roll our own
       onRefresh={refresh}
       refreshing={refreshing}
       sections={sections}
       stickySectionHeadersEnabled={false}
-      // TODO: figure out why this is causing duplicate ids
-      // keyExtractor={(item, index) => String(item.id + index)}
+      keyExtractor={(item, index) => `${item.id}  ${index}`}
```

Comment:
do we want to use index here instead of just id?

Reply:
I think there could potentially be duplicates in a lineup, so added id just in case